stdlib/public/SDK/Dispatch/Time.swift

```diff
@@ -109,18 +109,29 @@ public func ==(a: DispatchWallTime, b: DispatchWallTime) -> Bool {
 	return a.rawValue == b.rawValue
 }
 
-public enum DispatchTimeInterval {
+public enum DispatchTimeInterval: Equatable {
 	case seconds(Int)
 	case milliseconds(Int)
 	case microseconds(Int)
 	case nanoseconds(Int)
+	case never
```

Comment:
This is a source-breaking change; anyone that was switching over DispatchTimeInterval now has to worry about a new case. We should think a little harder before adding this.